bin/gobblin-mapreduce.sh

```diff
@@ -4,6 +4,9 @@
 ############### Run Gobblin Jobs on Hadoop MR ################
 ##############################################################
 
+# Set during the distribution build
+GOBBLIN_VERSION=@project.version@
```

Comment:
Ah I see. That makes much more sense. In that case, is it possible to make a few more additions to `bin/gobblin-mapreduce.sh` to ensure backwards compatibility:

* `bin/gobblin-mapreduce.sh` accepts a `projectVersion` parameter
    * If specified, it uses this as the `GOBBLIN_VERSION`
    * If not specified, it uses whatever has already been set for `GOBBLIN_VERSION`
        * If a valid `GOBBLIN_VERSION` has been set the script proceeds as usual
        * If a non-valid `GOBBLIN_VERSION` has been set - e.g. if it is still `@project.version@` then it should fail saying no `projectVersion` has been set.